docs/reference/schema.md

```diff
@@ -0,0 +1,170 @@
+# Schema reference
+
+The schema provides the authoritative definition of the structure of Open Fibre Data Standard (OFDS) data, the meaning of each field and the rules that must be followed to publish OFDS data. It is used to validate the structure and format of OFDS data.
+
+For this version of OFDS, the canonical URL of the schema is [https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-schema.json](https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-schema.json). Use the canonical URL to make sure that your software, documentation or other resources refer to the specific version of the schema with which they were tested.
+
+This page presents the schema in an [interactive browser](#browser) and in [reference tables](#reference-tables) with additional information in paragraphs. You can also download the canonical version of the schema as [JSON Schema](../../schema/network-schema.json) or download it as a [CSV spreadsheet](../../schema/network-schema.csv).
+
+```{note}
+   If any conflicts are found between the text on this page and the text within the schema, the text within the schema takes precedence.
+```
+
+## Browser
+
+Click on schema elements to expand the tree, or use the '+' icon to expand all elements. Use { } to view the underlying schema for any section. Required fields are indicated in **bold**.
+
+ <script src="../../_static/docson/widget.js" data-schema="../../_static/network-schema.json"></script> 
+
+## Reference tables
+
+This section presents each field in the schema in tables with additional information in paragraphs. Required fields are indicated in the **Required** column. For fields that reference components, a link is provided to a table with details of the component.
+
+### Structure
+
+This section describes the overall structure of the OFDS schema. The top-level object in OFDS data is a `Network`. A network has the following sections:
+
+* [Nodes](#nodes)
+* [Links](#links)
+* [Phases](#phases)
+* [Organisations](#organisations)
+* [Contracts](#contracts)
+
+In addition to the above sections, there are several top-level metadata fields:
```

Comment:
I think we want to keep the table in some form so that there is a tabular view of the properties at the top-level of the schema. Maybe we can just get rid of the bullet points and introduce the table with some extra framing.

My experience from OCDS is that some people prefer to use the browser and some people prefer to read the tables, so I think we should make sure that all the fields are available in each.